packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -315,13 +319,18 @@ export default compose( [
 			? selectedBlockIndex + 1
 			: endOfRootIndex;
 
+		const blockDoNotSupportInserter =
```

Comment:
ok I do not have strong opinion about the original name that I used. However before I change this check comment below because we also can get rid of all changes in the `inserter/index.js` file if we go with slightly different approach

Reply:
resolved according to [this](https://github.com/WordPress/gutenberg/pull/19013/files#r403054817)